### PR TITLE
Added playout delay extension support

### PR DESCRIPTION
**Pull request by sbarrac**
Adds support for the [playout delay extension](https://webrtc.googlesource.com/src/+/refs/heads/main/docs/native-code/rtp-hdrext/playout-delay)

### CODE REVIEW

**Comment by sbarrac** [comment]
Just thought I'd add some context.

This extension is really useful for cloud gaming scenarios. I was having an issue where the browser (chrome) would try to make the video smoother by delaying the rendering of a H264 stream. Even with ~10ms latency and zero packet loss, it was delaying the rendering by ~40MS, which is a huge delay in the context of playing a streamed game. 

The playout delay extension with minimum & maximum playout set to zero was the only way I found to remove this delay.

File: src/rtppacketizer.cpp [hunk 1]
```diff
@@ -85,6 +90,21 @@ message_ptr RtpPacketizer::packetize(shared_ptr<binary> payload, bool mark) {
...
+			uint16_t max = rtpConfig->playoutDelayMax & 0xFFF;
+
+			// 12 bits for min + 12 bits for max
+			char data[] = {
+				static_cast<char>((min >> 4) & 0xFF), 
+				static_cast<char>(((min & 0xF) << 4) | ((max >> 8) & 0xF)),
+				static_cast<char>(max & 0xFF)
+			};
+
+			extHeader->writeOneByteHeader(offset, rtpConfig->playoutDelayId, (byte *)data, 3);
```
Comment (paullouisageneau) [inline src/rtppacketizer.cpp:105]: You should use `byte` directly so you don't need casts:
```suggestion
    byte data[] = {
				byte((min >> 4) & 0xFF), 
			    byte(((min & 0xF) << 4) | ((max >> 8) & 0xF)),
				byte(max & 0xFF)
			};

			extHeader->writeOneByteHeader(offset, rtpConfig->playoutDelayId, data, 3);
```

**Review by paullouisageneau — COMMENTED** [review]
It looks good, I'm only concerned about the header size in the packetizer.

**Comment by sbarrac** [comment]
I think this is ready.

I've tested with different min/max values and it's working as expected. Debug output also shows no issues.

**Review by paullouisageneau — APPROVED** [review]
It looks good, thank you!

**Pull request merged**